base_station/gui/src/components/LCMEcho.vue

```diff
@@ -93,8 +93,9 @@
           '/set_demand': false,
           '/target_list': false,
           '/temperature': false,
-          '/tennis_ball': false
-
+          '/tennis_ball': false,
+          '/zed_gimbal_command': false,
```

Comment:
change to zed_gimbal_cmd to match the rest of our naming scheme

Reply:
What's the reasoning behind that change? `zed_gimbal_data` isn't as descriptive.

Reply:
it fits the naming convention, also in our icd it's 'zed_data' and 'zed_cmd'. 'zed_gimbal_cmd' and 'zed_gimbal_data' make more sense to me though upon seeing this

Reply:
I didn't see an interface doc in nav's directory on the drive. It must live somewhere else. I'll update that doc to match this. Since this is the first change in this area, it's best to make the naming better now rather than later.

Reply:
it should be in the nav ICD folder --- it is linked in the ESW one an the original copy is in the general ICD folder where all subteams' ICDs can be found

Reply:
I would prefer to stick with the data and cmd naming scheme when a cmd is being send vs data it's MUCH easier to debug if all the channels follow it

Reply:
Can you elaborate? I'm happy to change this to make things easier but none of our tools allow for searching like this and I don't know of a plan to add that anytime soon so I'd prefer to prioritize readability until then.

Reply:
searching like this? 

when debugging on the jetson to test things usually we don't use the gui bc it's not integrated yet so we'll be using lcm send/echo into the terminal. everything else was changed last year to follow the *name*_cmd or *name*_data scheme so it's annoying when something doesn't match, then we have to go looking for the specific struct/channel names 

also i think the decision we came to at a software leads meeting was actually to just go forward keeping those suffixes 

so i would rather keep it consistent

Reply:
Okay. I'll change it but would you still please try to answer why this helps? I'm curious why it matters since we can't search for `*_cmd` or `*_data`.

Reply:
its not searching for that it's when typing in the channel for lcm echo/send